WcaOnRails/app/models/wcif.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Wcif < ApplicationRecord
+  belongs_to :competition
+
+  # See https://github.com/thewca/worldcubeassociation.org/wiki/wcif
+  def json
```

Comment:
One idea might be to rename the database column to `json_str`, so it's clear that this is a string, and then continue to have `def json` and a `def json=(new_json)` methods that more clearly interact with the `json_str` database column.